### PR TITLE
Golang Homework 7.2 - TCP

### PR DESCRIPTION
**Pull request by stevenstr**
**Home work 7.2 - Task:**
• TCP Server (S) listen for a connections on a port
• TCP Client (C) connects to a TCP server on a port
• C reads STDIN for any input. On hit enter (‘\n’) C sends input it got to a server
• S reads input (split by ‘\n’) and checks if it’s an int, returns input multiplied by 2
• If it’s not an integer, return uppercased input string(floats will be untouched).
• C shows response from S to STDOUT and waits for another input from STDIN.
• C exits by input `exit`
• S exits by ctrl+C
 *** To connection I using PuTTY**

### CODE REVIEW

**Review by wshaman — APPROVED** [review]
It's not by the letter of the task. It should be a TCP client too.
But it's OK to get enough about understanding the topic.

File: main.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+		}
+
+		a, err := strconv.ParseInt(text, 10, 64)
+
+		if err == nil {
+			n := strconv.Itoa(int(a * int64(2)))
+			fmt.Println("Your number * 2 = ", n)
+			conn.Write([]byte("Success Multiply" + "\n\r"))
+
+		} else if err != nil {
```
Comment (burov) [inline main.go]: else if can be omitted just use ```continue``` or ```break``` try to avoid ```else if ``` and ```else``` where it's possible

**Review by burov — CHANGES_REQUESTED** [review]
I don't see a client, as well a few improvements possible

File: main.go (hunk 1 above)
Comment (stevenstr) [reply main.go]: All fix. Check it please!!!

**Review by burov — APPROVED** [review]
Good job :)

**Pull request closed without merging**